static/js/profile-galleries.js

```diff
@@ -0,0 +1,27 @@
+$(function() {
```

Comment:
I know that's a Python course, but since you also used JS, for your information, in JS the naming convention for variables and functions is `camelCase`.